aptos-node/src/lib.rs

```diff
@@ -190,8 +190,8 @@ pub fn start(
             .expect("Failed to build rayon global thread _pool.");
     }
 
-    let mut logger = aptos_logger::Logger::new();
-    logger
+    let mut logger_builder = aptos_logger::Logger::new();
```

Comment:
nit: use builder() instead of new() ?  But honestly I don't understand why we have both